classes/01class/exercises/c01-git03/feernandobraga/README.md

```diff
@@ -0,0 +1,23 @@
+# c01-git03
+
+## Questions (answer inline)
+
+1. Did you notice any difference in the size of the repo before and after adding the big file?
+   Yes, once I added the big file, the size jumped from a few KBs to 20M. However, even though I reverted the addition through git reset --hard HEAD^, the folder size was kept at 20M.
+
+2. What is the reason for this problem?
+   The reason for that is the command that I executed only pointed the HEAD to a previous position within the git history, but since git keeps a log/history of all my changes, the file was kept so I could also point my HEAD back to the position where/when the file existed, in case I wanted to revert my previous "git reset --hard HEAD^" action.
+
+3. How do you remove something from Git history after it is pushed to the remote repository? Which commands would you use?
+   To remove something from Git history, we need to re-write all commits starting from when the object(file.txt) we want to remove was added.
+   To do so, we use the following command:
+   git filter-branch --index-filter 'git rm --cached --ignore-unmatch file.txt' HEAD
```

Comment:
Hi Fernando, great work on the exercise. 

Dennis mentioned on another PR (#764 ) that ``filter-repo`` is recommended instead of ``filter-branch`` so you may want to change answer no 3 and 4 to reflect that :)

Reference: https://github.com/devopsacademyau/academy/pull/764#discussion_r443175317